pytest.ini

```diff
@@ -19,74 +19,51 @@ addopts =
     --ignore=docs/utils
 twisted = 1
+flake8-max-line-length = 99
 flake8-ignore =
     # Files that are only meant to provide top-level imports are expected not
     # to use any of their imports:
     scrapy/core/downloader/handlers/http.py F401
     scrapy/http/__init__.py F401
     # Issues pending a review:
-    # extras
-    extras/qps-bench-server.py E501
-    extras/qpsclient.py E501 E501
     # scrapy/commands
-    scrapy/commands/__init__.py E128 E501
-    scrapy/commands/check.py E501
-    scrapy/commands/crawl.py E501
-    scrapy/commands/edit.py E501
-    scrapy/commands/fetch.py E401 E501 E128 E502 E731
-    scrapy/commands/genspider.py E128 E501 E502
-    scrapy/commands/parse.py E128 E501 E731 E226
-    scrapy/commands/runspider.py E501
+    scrapy/commands/__init__.py E128
+    scrapy/commands/fetch.py E401 E128 E502 E731
+    scrapy/commands/genspider.py E128 E502
+    scrapy/commands/parse.py E128 E731 E226
     scrapy/commands/settings.py E128
-    scrapy/commands/shell.py E128 E501 E502
-    scrapy/commands/startproject.py E502 E127 E501 E128
-    scrapy/commands/version.py E501 E128
+    scrapy/commands/shell.py E128 E502
+    scrapy/commands/startproject.py E502 E127 E128
+    scrapy/commands/version.py E128
     # scrapy/contracts
-    scrapy/contracts/__init__.py E501 W504
+    scrapy/contracts/__init__.py W504
     scrapy/contracts/default.py E502 E128
     # scrapy/core
-    scrapy/core/engine.py E501 E128 E127 E306 E502
-    scrapy/core/scheduler.py E501
-    scrapy/core/scraper.py E501 E306 E128 W504
-    scrapy/core/spidermw.py E501 E731 E502 E126 E226
-    scrapy/core/downloader/__init__.py E501
-    scrapy/core/downloader/contextfactory.py E501 E128 E126
-    scrapy/core/downloader/middleware.py E501 E502
-    scrapy/core/downloader/tls.py E501 E305 E241
-    scrapy/core/downloader/webclient.py E731 E501 E502 E128 E126 E226
-    scrapy/core/downloader/handlers/__init__.py E501
-    scrapy/core/downloader/handlers/ftp.py E501 E305 E128 E127
-    scrapy/core/downloader/handlers/http10.py E501
+    scrapy/core/downloader/contextfactory.py E128 E126
+    scrapy/core/downloader/handlers/ftp.py E305 E128 E127
+    scrapy/core/downloader/handlers/s3.py E502 E128 E126
     scrapy/core/downloader/handlers/http11.py E501
-    scrapy/core/downloader/handlers/s3.py E501 E502 E128 E126
+    scrapy/core/downloader/middleware.py E502
+    scrapy/core/downloader/tls.py E305 E241
+    scrapy/core/downloader/webclient.py E731 E502 E128 E126 E226
+    scrapy/core/engine.py E128 E127 E306 E502
+    scrapy/core/scraper.py E306 E128 W504
+    scrapy/core/spidermw.py E731 E502 E126 E226
     # scrapy/downloadermiddlewares
-    scrapy/downloadermiddlewares/ajaxcrawl.py E501 E226
-    scrapy/downloadermiddlewares/decompression.py E501
-    scrapy/downloadermiddlewares/defaultheaders.py E501
-    scrapy/downloadermiddlewares/httpcache.py E501 E126
+    scrapy/downloadermiddlewares/ajaxcrawl.py E226 E501
```

Comment:
is this `E501` in the `scrapy/downloadermiddlewares/ajaxcrawl.py` for some reason?

Reply:
No reason, it must have slipped out. Updated, thanks for noticing!